apps/frontend/src/app/2_molecules/WalletBalance/WalletBalance.tsx

```diff
@@ -6,17 +6,29 @@ import { SimpleTable, SimpleTableRow } from '@sovryn/ui';
 import styles from './WalletBalance.module.css';
 
 export const WalletBalance: FC = () => {
-  const tokenOptions = useMemo(() => {
-    return SupportedTokenList.map(token => (
-      <SimpleTableRow
-        className={styles.row}
-        key={token.symbol}
-        label={token.symbol}
-        value="0" //TODO get balance from the wallet, task SOV-1101
-      />
-      //TODO: hardcoded, need to remove filter once we have a full list of tokens and ask Gilad to provide token symbols
-    )).filter(token => token.props.label !== SupportedTokens.sov);
-  }, []);
+  const tokensToDisplay = useMemo(
```

Comment:
Move it outside the component (can stay in this file), it doesn't make sense to have a `useMemo` with 0 dependencies inside a component.